### DISCUSSION TITLE
Week 2 Project Ideas

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#54] [General]
Project Ideas
1. Instructional webpage that teaches a lesson on story elements (plot, characters, setting, theme, etc.). Target audience: middle schoolers.
2. Create a website portfolio for writing and designs. 
3. Create an informative web page about a herbal business that is starting up. 
4. I like the idea of creating a new challenge here or elsewhere: https://css-challenges.com/ But I haven’t done anything with CSS yet.
5. Adding a lesson to Exercism sounds interesting mostly because this week I discovered that their instructions for the beginning tutorials/lessons are not spelled out well for beginners with no experience with JavaScript.
6. A webpage that teaches a lesson on what technical communication is. People are always asking me.

**Comment by metasean** [comment]
- 1, 2, 3, and 6
  - These are relatively safe, utilitarian projects.
  - You can make each of them as easy or challenging as you want.
  - You can also make each of them as vanilla or creative as you want
    - e.g. https://enhancv.com/blog/the-23-most-creative-resume-designs-weve-seen/

- CSS Challenges
  - By the end of the week, you'll have a bit more CSS experience. 

- Exercism
  - You might want to peruse [Exercism's Task list](https://exercism.org/contributing/tasks?track_slug=javascript&order=newest) and get an idea of what might be involved.
